index.html

```diff
@@ -107,6 +110,7 @@
                 .then(response => {
                     if (response.status !== 200) {
                         reject(new Error("Error!"));
+                        //
```

Comment:
Add a notification displayed to the user that the ajax call failed and data was not loaded correctly